.chloggen/supervisor-last-health-datarace.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: cmd/opampsupervisor
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Fix data race on access to `Supervisor.lastHealthFromClient`
```

```suggestion
note: Fix race condition where the Supervisor could report the wrong health status
```
I think we should reword this so it's targeted toward users, who won't necessarily be familiar with the code.